WorkingWithText/Program.cs

```diff
@@ -29,7 +76,31 @@ public static bool AreThereDuplicates(string hyphenNum)
         // consider it as False. Make sure that its returns false if any letters are passed.
         public static bool IsValidTime(string hyphenNum)
         {
- 			return default;
+            if (string.IsNullOrWhiteSpace(hyphenNum))
+                return false;
+            else if (hyphenNum[0] == '0' || hyphenNum[0] == '1' || hyphenNum[0] == '2')
+            {
+                if (hyphenNum.Contains(':'))
+                {
+                    var times = hyphenNum.Split(':');
+                    var hour = Convert.ToInt32(times[0]);
+                    var min = Convert.ToInt32(times[1]);
+
+                    if (times.Length != 2)
+                        return false;
+                    else if (hour < 0 || hour > 23)
+                        return false;
+                    else if (min < 0 || min > 59)
+                        return false;
+                    else
+                        return true;
+                }
+                else
+                    return false;
+            }
+            else
+                return false;
```

Comment:
A biiiiitttttt too many return statements here. I would recommend refactoring this to only have 1 or 2 returns.

Returns act as exit points. There are some programmers who say things like "Each method should only have one exit point" and having more than one can make them angry.

See this link,
https://stackoverflow.com/questions/36707/should-a-function-have-only-one-return-statement#:~:text=Structured%20programming%20says%20you%20should,functions%20with%20multiple%20return%20statements.

You can get away with 2 or maybe 3 in a small method like this, but 7 is too much.